execution_engine/src/storage/global_state/lmdb.rs

```diff
@@ -266,6 +267,29 @@ mod tests {
         ]
     }
 
+    fn create_n_test_pairs(n: usize) -> Vec<TestPair> {
+        let mut vec = Vec::with_capacity(n);
+
+        for i in 1..=n {
+            let key = {
+                let u256_val = U256::from(i);
+                let mut account_hash_bytes: [u8; 32] = Default::default();
+                u256_val.to_big_endian(&mut account_hash_bytes);
```

Comment:
Out of curiosity: we use little endian everywhere so why did you choose to use big endian here?